Frameworks/UIKit/UIPopoverPresentationController.mm

```diff
@@ -124,6 +122,17 @@ - (void)_dismissAnimated:(BOOL)animated completion:(dispatch_block_t)dismissComp
     [_popoverControllerInternal _dismissPopoverAnimated:animated completion:dismissCompletion];
 }
 
+- (void)_handleDismissCleanup {
+    // stay alive while UIViewController forgets about us
+    [self retain];
```

Comment:
>[self retain]; [](start = 4, length = 14)

Perhaps cleaner to just do:

StrongId<UIPopoverPresentationController> strongSelf = self; 

...so we don't have to manually call retain/release.